### PR TITLE
GPO Names and Behaviour Has Changed

### PR DESCRIPTION
**Pull request by brianreidc7**
"Enable automatic MDM enrollment using default Azure AD credentials" is the GPO name and it has two sub options. Not sure if Device Certificate is working at the moment, but the pictures are wrong, but User Certificate is working and so the docs should at least say to use that for now

### CODE REVIEW

File: windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy.md [hunk 1]
```diff
@@ -153,18 +153,16 @@ Requirements:
 - Enterprise AD must be integrated with Azure AD.
 - Ensure that PCs belong to same computer group.
 
-> [!IMPORTANT]
-> If you do not see the policy, it may be because you don’t have the ADMX installed for Windows 10, version 1803 or version 1809. To fix the issue, follow these steps:        
+>[!IMPORTANT]
+>If you do not see the policy, it may be because you don’t have the ADMX installed for Windows 10, version 1803 or version 1809. To fix the issue, follow these steps:
```
Comment (illfated) [inline windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy.md]: From what I know, the single space after the MarkDown quote markers should be there for compatibility when migrating the content to HTML. Do you have any reference stating that this spacing should be removed?

File: windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy.md [hunk 2]
```diff
@@ -106,7 +106,7 @@ Requirements:
 
     ![MDM policies](images/autoenrollment-mdm-policies.png)
 
-4. Double-click **Auto MDM Enrollment with AAD Token**.
+4. Double-click **Enable automatic MDM enrollment using default Azure AD credentials** (previously called **Auto MDM Enrollment with AAD Token** in version 1709 of Windows 10). For ADMX files from version 1903 and later select **User Credential** (support for Device Credential is coming) as the Selected Credential Type to Use. User Credential enrolls Windows 10 1709 and later once an Intune licenced user logs into the device. Device Credential will enroll the device and then assign a user later once support for this is available.
```
Comment (illfated) [inline windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy.md]: Readability (grammar) improvements:
```suggestion
4. Double-click **Enable automatic MDM enrollment using default Azure AD credentials** (previously called **Auto MDM Enrollment with AAD Token** in version 1709 of Windows 10). For ADMX files from version 1903 and later, select **User Credential** (support for Device Credential is coming) as the Selected Credential Type to use. User Credential enrolls Windows 10 1709 and later, once an Intune licensed user logs into the device. Device Credential will enroll the device and then assign a user later, once support for this is available.
```

File: windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy.md [hunk 3]
```diff
@@ -175,7 +173,6 @@ Requirements:
 5. Enforce a GPO link.
 
 ## Troubleshoot auto-enrollment of devices
-
```
Comment (illfated) [inline windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy.md]: Please keep the blank line spacing between this heading and its following text.

File: windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy.md [hunk 4]
```diff
@@ -229,9 +225,9 @@ To collect Event Viewer logs:
 - [Link a Group Policy Object](https://technet.microsoft.com/library/cc732979(v=ws.11).aspx)
 - [Filter Using Security Groups](https://technet.microsoft.com/library/cc752992(v=ws.11).aspx)
 - [Enforce a Group Policy Object Link](https://technet.microsoft.com/library/cc753909(v=ws.11).aspx)
+- [Group Policy Central Store](https://support.microsoft.com/en-gb/help/3087759/how-to-create-and-manage-the-central-store-for-group-policy-administra)
 
 ### Useful Links
 
 - [Windows 10 Administrative Templates for Windows 10 May 2019 Update 1903](https://www.microsoft.com/download/details.aspx?id=58495)
-- [Windows 10 Administrative Templates for Windows 10 October 2018 Update 1809](https://www.microsoft.com/download/details.aspx?id=57576)
 - [Windows 10 Administrative Templates for Windows 10 April 2018 Update 1803](https://www.microsoft.com/download/details.aspx?id=56880)
```
Comment (illfated) [inline windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy.md]: Out of curiosity, is there an advantage in removing the link to the 1809 ADMX files while also keeping the 1803 link available?
(I suppose the 1903 files are backwards compatible with 1809, but wouldn't that be the case for 1803 as well?)

Comment (brianreidc7) [reply windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy.md]: Incorrect edit by me. Elsewhere in the document 1803 was above 1809, so I just removed a line and added 1903 - thanks for spotting I removed the wrong line.

File: windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy.md [hunk 5]
```diff
@@ -148,24 +148,21 @@ Requirements:
...
+>   2. Install the package.
 >   3. Navigate, depending on the version to the folder:
->   1803 --> **C:\Program Files (x86)\Microsoft Group Policy\Windows 10 April 2018 Update (1803) v2**, or
->   1809 --> **C:\Program Files (x86)\Microsoft Group Policy\Windows 10 October 2018 Update (1809) v2** or
->   1903 --> **C:\Program Files (x86)\Microsoft Group Policy\Windows 10 May 2019 Update (1903) v3**
->   4. Copy policy definitions folder to **C:\Windows\SYSVOL\domain\Policies** .
->   5. Restart the Primary Domain Controller for the policy to be available.
+>   1903 --> **C:\Program Files (x86)\Microsoft Group Policy\Windows 10 May 2019 Update (1903) v3**, or  
+>   1809 --> **C:\Program Files (x86)\Microsoft Group Policy\Windows 10 October 2018 Update (1809) v2**
+>   4. Copy policy definitions folder to **C:\Windows\SYSVOL\domain\Policies** or **%windir%\sysvol\domain_name\policies\PolicyDefinitions** if an Group Policy Central Store exists.
```
Comment (illfated) [inline windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy.md]: Tiny grammar typo (an Group Policy), addition of "the"
```suggestion
>   4. Copy the policy definitions folder to **C:\Windows\SYSVOL\domain\Policies** or **%windir%\sysvol\domain_name\policies\PolicyDefinitions** if a Group Policy Central Store exists.
```

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by ManikaDhiman** [comment]
hi @e0i, this branch has merge conflics that must be resolved prior to merging. FYI, PR #4969 also updates the same section that is being updated via this PR. Changes per PR #4969 will be live shortly. Please review and compare the updates done there to resolve the merge conflict. Thanks!

**Comment by e0i** [comment]
@brianreidc7 Please resolve merge conflicts before we can move forward with this PR. Thanks.

File: windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy.md [hunk 6]
```diff
@@ -239,9 +235,9 @@ To collect Event Viewer logs:
 - [Link a Group Policy Object](https://technet.microsoft.com/library/cc732979(v=ws.11).aspx)
 - [Filter Using Security Groups](https://technet.microsoft.com/library/cc752992(v=ws.11).aspx)
 - [Enforce a Group Policy Object Link](https://technet.microsoft.com/library/cc753909(v=ws.11).aspx)
+- [Group Policy Central Store](https://support.microsoft.com/en-gb/help/3087759/how-to-create-and-manage-the-central-store-for-group-policy-administra)
```
Comment (illfated) [inline windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy.md]: I think we should let the doc links be language agnostic, even if the page in question might only be available in English.
```suggestion
- [Group Policy Central Store](https://support.microsoft.com/help/3087759/how-to-create-and-manage-the-central-store-for-group-policy-administra)
```

**Comment by e0i** [comment]
@ManikaDhiman The merge conflicts have been resolved.

@brianreidc7 Would you be able to follow-up with the minor revision suggestion provided by *illfated*? Thakns.

**Comment by e0i** [comment]
PR noted for reproduction without the conflicts.

**Comment by illfated** [comment]
Unless @brianreidc7 returns to this PR and resolves the merge conflict, I volunteer to open a replacement PR to apply the same changes as in this PR, based on the current state & content of the target page.

**Comment by e0i** [comment]
> Unless @brianreidc7 returns to this PR and resolves the merge conflict, I volunteer to open a replacement PR to apply the same changes as in this PR, based on the current state & content of the target page.

@illfated That would pretty much be appreciated. Thank you!

**Comment by brianreidc7** [comment]
I think I am done what you are asking - I could not see any references to en-gb in the actual doc though

**Comment by illfated** [comment]
Thank you for responding. What you are referring to might have been removed in other commits between October 2019 and now.

**Comment by e0i** [comment]
@ManikaDhiman The PR can be merged as it has passed copy review and merge conflicts have been resolved. Thank you.

cc @Dansimp

**Comment by martyav** [comment]
Hi, I'm a co-worker of Manika and on Dan's team. I was recently asked to help clean up the PR backlog.

**Pull request merged**